.github/workflows/build-plugin-archive.yml

```diff
@@ -168,32 +198,77 @@ jobs:
+        run: |
+          composer global config --no-plugins --no-interaction allow-plugins.inpsyde/composer-assets-compiler true
+          composer global require inpsyde/composer-assets-compiler
       - name: Run Composer Asset Compiler
         if: steps.composer-tools.outputs.assets-compiler == '0' && steps.check-for-package.outputs.has_package == 'true'
         run: composer compile-assets ${{ inputs.COMPILE_ASSETS_ARGS }}
 
+      - name: Install WordPress Translation Downloader
+        if: steps.composer-tools.outputs.translation-downloader == '0'
+        run: |
+          composer global config --no-plugins --no-interaction allow-plugins.composer/installers true
+          composer global config --no-plugins --no-interaction allow-plugins.inpsyde/wp-translation-downloader true
+          composer global require inpsyde/wp-translation-downloader
       - name: Run WordPress Translation Downloader
         if: steps.composer-tools.outputs.translation-downloader == '0'
         run: composer wp-translation-downloader:download
 
-      - name: Install Composer dependencies without dev dependencies
-        # ⬑ Now we want to get rid of dev dependencies and most probably skip composer scripts
-        uses: ramsey/composer-install@v3
+      - name: Set artifact name
+        id: set-artifact-name
+        run: echo "artifact=interim-built" >> $GITHUB_OUTPUT
+      - name: Upload artifact
+        uses: actions/upload-artifact@v4
         with:
-          composer-options: ${{ inputs.COMPOSER_ARGS }}
+          name: ${{ steps.set-artifact-name.outputs.artifact }}
+          path: |
+            .
+            !**/node_modules
+  create-plugin-archive:
+    name: Create build archive
+    timeout-minutes: 5
+    runs-on: ubuntu-latest
+    needs: run-build-tools
+    env:
+      ARCHIVE_NAME: ${{ inputs.ARCHIVE_NAME }}
+      PLUGIN_FOLDER_NAME: ${{ inputs.PLUGIN_FOLDER_NAME }}
+      GIT_SHA: ${{ github.sha }}
+      PLUGIN_VERSION: ${{ inputs.PLUGIN_VERSION }}
+    steps:
+      - name: Download Artifact
+        uses: actions/download-artifact@v4
+        with:
+          name: ${{ needs.run-build-tools.outputs.artifact }}
+
+      - name: Set up plugin folder name
+        id: plugin-folder-name
+        run: echo "plugin-folder-name=${PLUGIN_FOLDER_NAME:-${ARCHIVE_NAME:-${{ github.event.repository.name }}}}" >> $GITHUB_OUTPUT
+
+      - name: Set up archive name
+        id: plugin-data
+        run: echo "archive-name=${ARCHIVE_NAME:-${{ github.event.repository.name }}}" >> $GITHUB_OUTPUT
 
       - name: Add commit hash to plugin header
         run: 'sed -Ei "s/SHA: .*/SHA: ${GIT_SHA}/g" ${{ inputs.PLUGIN_MAIN_FILE }}'
 
       - name: Set plugin version header
         run: 'sed -Ei "s/Version: .*/Version: ${PLUGIN_VERSION}/g" ${{ inputs.PLUGIN_MAIN_FILE }}'
 
+      - name: Set up PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: '8.3'
```

Comment:
Let's please not hardcode a PHP version here. I understand you want to use a different version than `${{ inputs.PHP_VERSION }}`, but in this case, please create an additional input (which ideally would default to PHP 8.0 to be aligned with all other PHP version inputs; see #108).

Reply:
Ha! I was actually just trying to see if the `tools:` section would make life easier by setting up `php-scoper` in the `PATH` as opposed to my previous hamfisted approach of finding the executable.
I even recall thinking "maybe this should be a new input" but then forgot. I'll add this

Reply:
Done